src/components/PlanPage/PlanPage.js

```diff
@@ -11,15 +12,25 @@ class PlanPage extends React.Component {
     onStartSetActivities: PropTypes.func.isRequired,
   };
 
-  componentDidMount() {
-    this.props.onStartSetActivities();
+  constructor(props) {
+    super(props);
+    this.state = {
+      loaded: false,
```

Comment:
it is nice convention (I am pretty sure, that we've been talking aobut that) to name boolean values as `isLoaded`, `hasShadow`, etc.

`bulma.io`